src/10_functions.py

```diff
@@ -1,6 +1,12 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(num):
```

Comment:
You combined both functions into 1 here. What the assignment was looking for was 2 separate functions
```

def is_even(num):
    if num % 2 == 0:
        return True
```
```

def answer(num):
    if is_even(num) is True:
        print("EVEN")
    else:
        print("ODD")


answer(num)
```